src/sidebar.jsx

```diff
@@ -352,7 +313,7 @@ const DropdownWithKebab = ({
     const multiDropdownOptions = [
         {
             id: "copy-item",
-            onClick: () => copyItem(setClipboard, selected.map(s => path.join("/") + "/" + s.name)),
+            onClick: () => copyItems(setClipboard, selected.map(s => path.join("/") + "/" + s.name)),
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-254-20240212-112317-82181124-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)